woocommerce/google-listings-and-ads
# Add bcmath compatibility library

### Changes proposed in this Pull Request:

We already have `phpseclib` and `phpseclib/bcmath_compat` is quite small so I added it as a compatibility library for cases where the PHP extension `bcmath` is not installed. The Google protobuf library requires the `bccomp` function to be present.

Closes #594 

### Detailed test instructions:

1. Configure PHP to skip loading of the bcmath extension `extension=bcmath.so`
2. View the Google Listings & Ads > Dashboard page
3. Confirm that we get a fatal error
4. Apply this PR and run composer install to include the compat library
5. View the Dashboard page again and confirm this time it loads without errors

### Changelog Note:
* Fix - Add bcmath compatibility library.
